src/commands/appSettings/connectionSettings/sqlDatabase/SqlDatabaseConnectionPromptStep.ts

```diff
@@ -23,31 +21,17 @@ export class SqlDatabaseConnectionPromptStep<T extends ISqlDatabaseConnectionWiz
     public async prompt(context: T): Promise<void> {
         const connectAzureDatabase: MessageItem = { title: localize('connectSqlDatabase', 'Connect Azure SQL Database') };
         const connectNonAzureDatabase: MessageItem = { title: localize('connectSqlDatabase', 'Connect Non-Azure SQL Database') };
-        const useExistingConnectionButton: MessageItem = { title: localize('useExistingConnection', 'Use Existing Connection') };
-        const skipForNowButton: MessageItem = { title: skipForNow };
 
         const message: string = localize('selectSqlDatabaseConnection', 'In order to proceed, you must connect a SQL database for internal use by the Azure Functions runtime.');
 
         const buttons: MessageItem[] = [connectAzureDatabase, connectNonAzureDatabase];
 
-        if (!this._options?.suppressSkipForNow) {
-            buttons.push(skipForNowButton);
-        } else if (this._options?.suppressSkipForNow && !context.sqlDbRemoteConnection) {
```

Comment:
We've gone back and forth on this one a bit, but I think the decision you and I made before New Years was to just keep the behavior similar to the way we had it before - that is, only prompt when the local settings connection string field is empty, otherwise don't prompt.

Also I am currently only supporting `Azure` and `Custom` for Sql.  In a recent commit I changed `NonAzure` to be split up into `Emulator` and `Custom`.  There's a lot more omit/exclude logic, hopefully it makes the scenarios a bit more clear 🤔